tests/test-wp-parsely.php

```diff
@@ -193,24 +193,7 @@ public function setUp() {
 		);
 		update_option( 'parsely', $option_defaults );
 		self::$parsely_html = <<<PARSELYJS
-<div id="parsely-root" style="display: none">
-	<div id="parsely-cfg" data-parsely-site="blog.parsely.com"></div>
-</div>
-<script data-cfasync="false">
```

Comment:
we need to keep the data-cfasync thing, or cloudfront will kablammo our script

Reply:
ah, totally missed that, thanks for the catch.